packages/p2p/src/pages/my-ads/my-ads-row-renderer.jsx

```diff
@@ -321,7 +327,7 @@ const MyAdsRowRenderer = observer(({ country_list, row: advert, table_ref }) =>
     }
 
     return (
-        <div ref={row_ref}>
+        <div ref={row_ref} id={`my_ads_table_row_${id}`}>
```

Comment:
what is this id for?

Reply:
its so the dropdown knows where to portal to,I added it above in the dropdown, this is only for desktop. Reason i did this, was so it would make sure its scoped to to row itself, cause it was having issues when i didn't use this